openstack/clientconfig/utils.go

```diff
@@ -99,7 +99,7 @@ func mergeInterfaces(overridingInterface, inferiorInterface interface{}) interfa
 // 4. unix-specific site_config_dir (/etc/openstack/clouds.yaml)
 //
 // If found, the contents of the file is returned.
-func FindAndReadCloudsYAML() (string, []byte, error) {
+func FindAndReadCloudsYAML() (s string, b []byte, err error) {
```

Comment:
nit: Can you revert this? I'm not opposed to named return values, but introducing this would break style consistency within the rest of the code.

Reply:
Done!